src/main/resources/application.properties.EXAMPLE

```diff
@@ -403,4 +403,14 @@ server.max-http-request-header-size=16384
 ## Enable Study Help Button on Study View Page
 #skin.show_study_help_button=true
 
+# Sentry Config
+#sentry.dsn=
+#sentry.traces-sample-rate=1.0 Probably should change for prod
+#sentry.exception-resolver-order=-2147483647
+
+# Swagger Configuration
+springdoc.swagger-ui.disable-swagger-default-url=true
+springdoc.swagger-ui.path=/api/swagger-ui
+springdoc.api-docs.path=/api/v2/api-docs
```

Comment:
I feel slightly uncomfortable about making this the default in the EXAMPLE properties: springdoc.api-docs.path=/api/v2/api-docs. As I guess it is actually v3 of the api-docs. Might be cleaner if we can somehow serve both /api/v3/api-docs and /api/v2/api-docs . Wondering if in the future we can prevent this by just having e.g. a /api/api-docs. Anyway can be follow up ticket, filed on here: https://github.com/cBioPortal/cbioportal/issues/10534. Prolly ok for now